modules/serverless-pingsource-kn.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * serverless/knative_eventing/serverless-pingsource.adoc
+
+The following sections describe how to create, verify and remove a basic PingSource using the `kn` CLI.
+
+.Prerequisites
+
+* You have Knative Serving and Eventing installed.
+* You have the `kn` CLI installed.
+
+.Procedure
+
+. To verify that `PingSource` is working, create a simple Knative
+service that dumps incoming messages to its log.
++
+You can create the service by entering the following command:
++
+----
+$ kn service create event-display --image quay.io/openshift-knative/knative-eventing-sources-event-display:v0.13.2
+----
+
+. For each set of ping events that you want to request, create a PingSource in the same namespace as the event consumer.
++
+You can create a PingSource by entering the following command:
++
+----
+$ kn source ping create test-ping-source --schedule "*/2 * * * *" --data '{ value: "hello" }' --sink svc:event-display
```

Comment:
There's a different `--data` value in `yaml` section, that might be a better example, `'{"message": "Hello world!"}'`.

There's no functional difference, more a consistency issue with log and `kn describe` output. If you think it should be changed, I'll provide new outputs, np.

Reply:
Nope, that sounds good, this was just sort of my best guess at adding the command, happy to update to '{"message": "Hello world!"}' :)